packages/components/anchor/src/Anchor.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <component :is="wrapperComponent" :target="target" :offset="offsetTop">
+    <div class="ix-anchor-wrapper" :style="wrapperStyle">
+      <div class="ix-anchor">
+        <div class="ix-anchor-ink">
+          <span v-show="showInkInFixed" :class="inkCls" :style="{ top: `${inkTop}px` }" />
+        </div>
+        <slot></slot>
+      </div>
+    </div>
+  </component>
+</template>
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  onMounted,
+  onUnmounted,
+  watch,
+  ref,
+  nextTick,
+  provide,
+  Ref,
+  SetupContext,
+  unref,
+} from 'vue'
+import { AnchorProps, Section } from './types'
+import { PropTypes, on, off, isNil } from '@idux/cdk/utils'
+import { getOffsetTop, scrollTo, getCurrentScrollTop } from './util'
+import { getTarget } from '@idux/components/utils'
+
+export default defineComponent({
+  name: 'IxAnchor',
+  props: {
+    affix: PropTypes.bool.def(true), //固定模式, false为不浮动，状态不随页面滚动变化,默认true
+    showInkInFixed: PropTypes.bool.def(false), //固定模式是否显示小圆点,默认false
+    target: PropTypes.oneOfType([PropTypes.string, HTMLElement, Window]).def(() => window), //指定滚动的元素。默认window
+    offsetTop: PropTypes.number, //距离窗口顶部达到指定偏移量后触发
+    targetOffset: PropTypes.number, //锚点滚动偏移量，默认与 offsetTop 相同
+  },
+  emits: ['change', 'click'],
+  setup(props: AnchorProps, context: SetupContext) {
+    const eventType = 'scroll'
+    const wrapperComponent = computed(() => (props.affix ? 'IxAffix' : 'div'))
+    const container = ref<Window | HTMLElement>((null as unknown) as HTMLElement)
+
+    const { activeLink, registerLink, unregisterLink, links, setActiveLink } = useLinks()
+    const { inkCls, updateInk, inkTop } = useInk(props, activeLink)
+
+    const elOffsetTop = unref(computed(() => (!isNil(props.targetOffset) ? props.targetOffset : props.offsetTop || 0)))
```

Comment:
```suggestion
    const elOffsetTop = computed(() => (!isNil(props.targetOffset) ? props.targetOffset : props.offsetTop || 0))
```
这里写的有问题吧，这样会失去响应式特性。